src/Sets/Hyperrectangle.jl

```diff
@@ -71,18 +71,25 @@ julia> h = [1.0, 2.0];
 julia> Hyperrectangle(low=l, high=h)
 Hyperrectangle{Float64,Array{Float64,1},Array{Float64,1}}([-1.0, 1.0], [2.0, 1.0])
 ```
+
+By default, the constructor checks that that radius of the hyperrecatangle
```

Comment:
```suggestion
By default, the constructor checks that that radius of the hyperrectangle
```

Reply:
> hyperrecatangle

sounds funny :laughing:  we should define such type